airflow/models/taskinstance.py

```diff
@@ -1719,6 +1720,11 @@ def handle_failure(
             self.state = State.FAILED
             email_for_state = task.email_on_failure
         else:
+            if self.state == State.QUEUED:
+                # We increase the try_number so as
+                # to fail the task if it fails to start
+                # after sometime
+                self.try_number += 1
```

Comment:
```suggestion
                self._try_number += 1
```

Reply:
(This might not have any affect but is more correct -- as that way we don't have to reason about "has the try_number property already added 1 to this value or not")